hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -331,55 +331,55 @@ public long storeSize(String table) throws IOException {
         /**
          * Add a row record to a table
          */
-        public abstract void put(String table, byte[] family, byte[] rowkey,
-                                 Collection<BackendColumn> columns);
+        void put(String table, byte[] family, byte[] rowkey,
+                 Collection<BackendColumn> columns);
 
         /**
          * Add a row record to a table(can be used when adding an index)
          */
-        public abstract void put(String table, byte[] family,
-                                 byte[] rowkey, byte[] qualifier, byte[] value);
+        void put(String table, byte[] family, byte[] rowkey, byte[] qualifier,
+                 byte[] value);
 
         /**
          * Delete a record by rowkey and qualifier from a table
          */
-        public default void remove(String table, byte[] family,
-                                   byte[] rowkey, byte[] qualifier) {
+        default void remove(String table, byte[] family, byte[] rowkey,
```

Comment:
prefer to wrap line before "byte[] rowkey"

Reply:
this line only 72 char